tests/ValueParsers/IsoTimestampParserTest.php

```diff
@@ -130,6 +131,11 @@ public function validInputProvider() {
 				'+2000000000000000-00-00T00:00:00Z',
 				TimeValue::PRECISION_Ga,
 			),
+			'-2000000000000000-00-00T00:00:00Z' => array(
```

Comment:
Marking them as Gregorian doesn't make any more sense. It just doesn't matter. So, for consistency, it's easier to just assume anything before 1582 is Julian.

Astronomers use Julian years for dates in the deep past, as far as I know.